arcgis-ios-sdk-samples/Maps/Change viewpoint/SetViewpointViewController.swift

```diff
@@ -83,10 +83,10 @@ class SetViewpointViewController: UIViewController {
             let currentScale = self.mapView.mapScale
             let targetScale = currentScale / 2.5 //zoom in
             let currentCenter = self.mapView.visibleArea!.extent.center
-            self.mapView.setViewpoint(AGSViewpoint(center: currentCenter, scale: targetScale), duration: 5, curve: AGSAnimationCurve.EaseInOutSine, completion: { (finishedWithoutInterruption) -> Void in
+            self.mapView.setViewpoint(AGSViewpoint(center: currentCenter, scale: targetScale), duration: 5, curve: AGSAnimationCurve.easeInOutSine, completion: { (finishedWithoutInterruption) -> Void in
```

Comment:
not 100% sure we can use trailing closure syntax here, but please change if we can